spec/models/guide_spec.rb

```diff
@@ -60,115 +60,129 @@
     end
   end
 
-  describe "validations" do
-    it "ensures that slugs aren't saved without the topic name in the path" do
-      guide = Guide.new(slug: "/service-manual/guide-path")
-      guide.valid?
-      expect(guide.errors.full_messages_for(:slug)).to eq ["Slug must be present and start with '/service-manual/[topic]'"]
-    end
+  describe "#valid?" do
```

Comment:
The `describe "validations"` isn't essential but it's pretty common. The validations are also run when you call `#save`/ `#create`/`#update`. Neither is really wrong but I think most of this app does `describe "validations"` and talks about the thing it's validating in another nested block. eg.

```
describe "validations" do
  describe "slug" do
    it "does this"
    it "does that"
  end
end
```